apps/docs/self-hosting/get-started.mdx

```diff
@@ -98,5 +98,6 @@ I've written guides on how to deploy Typebot using:
 - [Docker](./deploy/docker)
 - [Vercel](./deploy/vercel)
 - [Manual](./deploy/manual)
+- [ALibaba Cloud](./deploy/alibaba-cloud)
```

Comment:
```suggestion
- [Alibaba Cloud](./deploy/alibaba-cloud)
```

Reply:
Hello brother, the first "l" in Alibaba Cloud should be lowercase because it is a proprietary term for our company. However, it is very understandable for this to happen, as at this stage, Alibaba Cloud's influence is mostly in the Asia-Pacific region.

Reply:
Isn't `ALibaba` a typo? Here, I suggest you just fix it to `Alibaba`

Reply:
Sorry, you are undoubtedly right, I must have seen it wrong.